dev-java/java-service-wrapper/java-service-wrapper-3.5.50.ebuild

```
@@ -47,7 +47,11 @@ src_prepare() {
 	cp "${S}/src/c/Makefile-linux-armel-32.make" "${S}/src/c/Makefile-linux-arm-32.make"
 	java-pkg-2_src_prepare
 
-	# enable tests on all platforms
+	# disable tests by default (they are only enabled by default on amd64)
+	sed "s/all: init wrapper libwrapper.so testsuite/all: init wrapper libwrapper.so/g" \
+		src/c/Makefile-linux-x86-64.make || die
```

Small improvement with shorter sed, and the very important `-i`!

```suggestion
	sed -e "s/\(all: init wrapper libwrapper.so\) testsuite/\1/g" \
		-i src/c/Makefile-linux-x86-64.make || die
```

---

Oops!!!  Good catch sorry!